src/browser/squirrel-update.coffee

```diff
@@ -231,19 +129,24 @@ exports.handleStartupEvent = (app, squirrelCommand) ->
   switch squirrelCommand
     when '--squirrel-install'
       createShortcuts ->
-        installContextMenu ->
+        WinRegistry.installContextMenu ->
           addCommandsToPath ->
             app.quit()
       true
     when '--squirrel-updated'
       updateShortcuts ->
-        installContextMenu ->
+        contextMenuWasEnabled = atom.config.get('core.showAtomInShellContextMenu')
```

Comment:
It worked during tests. We don't yet have a Windows machine at hand, so we did not actually run the build script (or whatever is needed) on Windows.